### DISCUSSION TITLE
Where are docs appendix?

### DISCUSSION THREAD

**Discussion by mmaridev** [osm-search/Nominatim#2670] [Q&A] [answered]
Hi,

I wanted to submit a PR to add in the Ubuntu 20.04 guide the indication to set the locale before compiling and installing Nominatim, as I've found in another discussion here.

The link points to `../appendix/Install-on-Ubuntu-20.md` which doesn't exist in git. Where are this files?

**Comment by mtmail** [comment, marked as answer]
https://github.com/osm-search/Nominatim/blob/master/vagrant/Install-on-Ubuntu-20.sh

There's a bit of custom logic in https://github.com/osm-search/Nominatim/blob/master/docs/CMakeLists.txt which compiles the output into the https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-20/ URL

**Reply by mmaridev** [reply]
Perfect, thank you!
